tests/includes/wait-for.sh

```diff
@@ -172,6 +172,39 @@ wait_for_machine_agent_status() {
 	fi
 }
 
+# wait_for_container_agent_status blocks until the machine agent for the specified
+# machine instance ID reports the requested status.
+#
+# ```
+# wait_for_container_agent_status <parent-id> <instance-id> <status>
+#
+# example:
+# wait_for_container_agent_status "0" "0/lxd/0 "started"
```

Comment:
The machine is superfluous. It can be derived from the container id.

Reply:
fixed